### PR TITLE
VSCode_Query_Breakdown: Project Initialized 2 

### PR DESCRIPTION
**Pull request by subinbean**
Boilerplate code for the VSCode Extension (with correct configurations). No logic is implemented yet. Also note two slight file changes that are irrelevant (InputReader/LocationTracker) that happened while solving merge conflicts

### CODE REVIEW

File: tools/query_breakdown/src/main/java/com/google/bigquery/InputReader.java [hunk 1]
```diff
@@ -10,7 +10,6 @@
  * into queries and data-cleaning if needed.
  */
 public class InputReader {
-
```
Comment (feiling) [inline tools/query_breakdown/src/main/java/com/google/bigquery/InputReader.java:13]: Pls remove this file from the pull request.

File: tools/query_breakdown/src/main/java/com/google/bigquery/LocationTracker.java [hunk 2]
```diff
@@ -95,5 +95,4 @@ public LocationTracker cloneTracker() {
     return locationTracker;
   }
 
-
```
Comment (feiling) [inline tools/query_breakdown/src/main/java/com/google/bigquery/LocationTracker.java:98]: Again, pls remove this file from the pull request.

File: tools/query_breakdown/src/main/java/com/google/bigquery/InputReader.java (hunk 1 above)
Comment (subinbean) [reply tools/query_breakdown/src/main/java/com/google/bigquery/InputReader.java]: removed!

File: tools/query_breakdown/src/main/java/com/google/bigquery/LocationTracker.java (hunk 2 above)
Comment (subinbean) [reply tools/query_breakdown/src/main/java/com/google/bigquery/LocationTracker.java]: removed!

**Pull request merged**